### PR TITLE
Fixed multiple documentation problems

### PR DESCRIPTION
**Pull request by dpgraham**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: commands-yml/commands/element/find-elements.yml [hunk 1]
```diff
@@ -5,14 +5,20 @@ description:
...
 selector_strategies:
-  # TODO Document the rest
   - name: Accessibility ID
     description: Read a unique identifier for a UI element. For XCUITest it is the element's `resource-id` attribute. For Android it is the element's `content-desc` attribute.
   - name: Class name
     description: "For IOS it is the full name of the XCUI element and begins with XCUIElementType. For Android it is the full name of the UIAutomator2 class (e.g.: android.widget.TextView)"
   - name: ID
-    description: Native element identifier. For Android 
-    
+    description: Native element identifier. For Android it is
```
Comment (imurchie) [inline commands-yml/commands/element/find-elements.yml]: This seems incomplete.

File: commands-yml/commands/element/find-elements.yml [hunk 2]
```diff
@@ -5,14 +5,20 @@ description:
...
+  - name: Name
+    description: Name of element
+  - name: XPath
+    description: Search the app XML source using xpath (not recommended, has performance issues)
+  - name: Android UiAutomator
+    description: Use the UI Automator API, in particular the UiSelector class to locate elements. In Appium you send the Java code, as a string, to the server, which executes it in the application’s environment, returning the element or elements.    
+  - name: IOS UIAutomation
+    description: When automating an iOS application, Apple’s UI Automation framework can be used to find elements
```
Comment (imurchie) [inline commands-yml/commands/element/find-elements.yml]: Only for iOS < 10, using Instruments-based automation. This is not available in xcuitest-based tests.

File: commands-yml/commands/element/find-elements.yml [hunk 3]
```diff
@@ -5,14 +5,20 @@ description:
...
   - name: ID
-    description: Native element identifier. For Android 
-    
+    description: Native element identifier. For Android it is
+  - name: Name
+    description: Name of element
+  - name: XPath
+    description: Search the app XML source using xpath (not recommended, has performance issues)
+  - name: Android UiAutomator
+    description: Use the UI Automator API, in particular the UiSelector class to locate elements. In Appium you send the Java code, as a string, to the server, which executes it in the application’s environment, returning the element or elements.
```
Comment (imurchie) [inline commands-yml/commands/element/find-elements.yml]: Is there a link to the API from Android?

**Review by imurchie — CHANGES_REQUESTED** [review]

**Comment by dpgraham** [comment]
Fixes pushed.

File: commands-yml/commands/element/find-elements.yml [hunk 4]
```diff
@@ -5,14 +5,20 @@ description:
...
-    
+    description: Native element identifier. `resource-id` for android; `name` for iOS.
+  - name: Name
+    description: Name of element
+  - name: XPath
+    description: Search the app XML source using xpath (not recommended, has performance issues)
+  - name: Android UiAutomator
+    description: Use the UI Automator API, in particular the [UiSelector](https://developer.android.com/reference/android/support/test/uiautomator/UiSelector.html) class to locate elements. In Appium you send the Java code, as a string, to the server, which executes it in the application’s environment, returning the element or elements.    
+  - name: IOS UIAutomation
+    description: When automating an iOS application, Apple’s UI Automation framework can be used to find elements
```
Comment (imurchie) [inline commands-yml/commands/element/find-elements.yml]: This still only works for Instruments-based automation.

Comment (dpgraham) [reply commands-yml/commands/element/find-elements.yml]: Fix made.

File: commands-yml/commands/element/find-elements.yml [hunk 5]
```diff
@@ -5,14 +5,20 @@ description:
...
-    
+    description: Native element identifier. `resource-id` for android; `name` for iOS.
+  - name: Name
+    description: Name of element
+  - name: XPath
+    description: Search the app XML source using xpath (not recommended, has performance issues)
+  - name: Android UiAutomator
+    description: Use the UI Automator API, in particular the [UiSelector](https://developer.android.com/reference/android/support/test/uiautomator/UiSelector.html) class to locate elements. In Appium you send the Java code, as a string, to the server, which executes it in the application’s environment, returning the element or elements.    
+  - name: IOS UIAutomation
+    description: When automating an iOS application, Apple’s UI Automation framework can be used to find elements (Instruments only)
```
Comment (imurchie) [inline commands-yml/commands/element/find-elements.yml]: I think this should be a link to the Instruments-based automation docs (https://github.com/appium/appium/blob/master/docs/en/drivers/ios-uiautomation.md)

File: commands-yml/commands/element/find-elements.yml [hunk 6]
```diff
@@ -5,14 +5,20 @@ description:
...
-    
+    description: Native element identifier. `resource-id` for android; `name` for iOS.
+  - name: Name
+    description: Name of element
+  - name: XPath
+    description: Search the app XML source using xpath (not recommended, has performance issues)
+  - name: Android UiAutomator
+    description: Use the UI Automator API, in particular the [UiSelector](https://developer.android.com/reference/android/support/test/uiautomator/UiSelector.html) class to locate elements. In Appium you send the Java code, as a string, to the server, which executes it in the application’s environment, returning the element or elements.    
+  - name: IOS UIAutomation
+    description: When automating an iOS application, Apple’s [Instruments](https://github.com/appium/appium/blob/master/docs/en/drivers/ios-uiautomation.md) framework can be used to find elements
```
Comment (imurchie) [inline commands-yml/commands/element/find-elements.yml]: This link needs to be relative or it will not work on appium.io.

**Review by imurchie — APPROVED** [review]

**Pull request merged**